### PR TITLE
Removed redundant `Doctrine\ORM\EntityRepository#__construct()` phpdoc

### PR DESCRIPTION
**Pull request by jvasseur**
Fix EntityRepository constructor phpdoc to match type declaration

### CODE REVIEW

File: lib/Doctrine/ORM/EntityRepository.php [hunk 1]
```diff
@@ -58,8 +58,8 @@ class EntityRepository implements ObjectRepository, Selectable
     /**
      * Initializes a new <tt>EntityRepository</tt>.
      *
-     * @param EntityManager         $em    The EntityManager to use.
```
Comment (Ocramius) [inline lib/Doctrine/ORM/EntityRepository.php]: Both of these docblock entries can be dropped, as they are redundant

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: lib/Doctrine/ORM/EntityRepository.php (hunk 1 above)
Comment (jvasseur) [reply lib/Doctrine/ORM/EntityRepository.php]: Yeah the descriptions aren't realy useful. I removed them.

**Review by Ocramius — DISMISSED** [review]
LGTM 👍

**Comment by Ocramius** [comment]
@jvasseur hmm, I just noticed that the patch targets `2.6`. can you point it at `master` please?

**Comment by jvasseur** [comment]
@Ocramius Done.

**Review by Ocramius — APPROVED** [review]
:+1:

**Review by Majkl578 — APPROVED** [review]

**Comment by Majkl578** [comment]
@Ocramius Forgot to hit merge? :stuck_out_tongue_winking_eye:

**Comment by Ocramius** [comment]
Nah, was waiting for the build, but no uplink while on the train today...

**Pull request merged**